pkg/library/container/conversion.go

```diff
@@ -79,9 +78,45 @@ func devfileResourcesToContainerResources(devfileContainer *dw.ContainerComponen
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse memory limit %q: %w", memLimit, err)
 	}
+
+	memReq := devfileContainer.MemoryRequest
+	if memReq == "" {
+		memReq = "512Mi"
+	}
+	if memReq == "" {
+		memReq = constants.SidecarDefaultMemoryRequest
+	}
```

Comment:
```suggestion
	if memReq == "" {
		memReq = constants.SidecarDefaultMemoryRequest
	}
```

Reply:
:man_facepalming: how it was possible create PR with such pure quality.
@amisevsk thanks for your catches.

Reply:
Double facepalm. I would found my mistakes earlier if I run tests :-D